tests/debugger/test_debugger_expression_language.py

```diff
@@ -2,34 +2,57 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-import tests.debugger.utils as base
+import tests.debugger.utils as debugger
 import re, json
-from utils import scenarios, interfaces, weblog, features, remote_config as rc, bug, context
+from utils import scenarios, features, bug, context
 
 
 @features.debugger_expression_language
 @scenarios.debugger_expression_language
-class Test_Debugger_Expression_Language(base._Base_Debugger_Test):
-    version = 0
+class Test_Debugger_Expression_Language(debugger._Base_Debugger_Test):
     message_map = {}
-    tracer = None
 
+    ############ setup ############
     def _setup(self, probes, request_path):
-        self.installed_ids = set()
-        self.expected_probe_ids = base.extract_probe_ids(probes)
+        self.set_probes(probes)
+        self.send_rc_probes()
+        self.wait_for_all_probes_installed()
+        self.send_weblog_request(request_path)
 
-        Test_Debugger_Expression_Language.version += 1
-        self.rc_state = rc.send_debugger_command(probes=probes, version=Test_Debugger_Expression_Language.version)
+    ############ assert ############
+    def _assert(self, expected_response: int):
```

Comment:
What was the reason for removing the code defaulting to 200 here? Seems like almost always this would be the desired code.

Reply:
Personally, I don't like default values and prefer to have code more explicit.